src/openms/include/OpenMS/METADATA/Software.h

```diff
@@ -50,14 +50,18 @@ namespace OpenMS
   {
 public:
     /// Constructor
-    explicit Software(const String& name = "", const String& version = "");
+    Software() = default;
     /// Copy constructor
-    Software(const Software& source) = default;
+    Software(const Software & source) = default;
+    /// Move constructor
+    Software(Software&&) = default;
     /// Destructor
     ~Software() override;
 
     /// Assignment operator
-    Software& operator=(const Software& source) = default;
+    Software & operator=(const Software & source) = default;
+    /// Move assignment operator
+    Software& operator=(Software&&) & = default;
```

Comment:
@hendrikweisser we are currently debating here how to handle these "simple" cases https://github.com/OpenMS/OpenMS/issues/3310#issuecomment-424066542 and we are torn between your approach (Rule of Five) and the cleaner "Rule of Zero"

Reply:
Some changes to `Software` got lost, presumably in a merge conflict. Let's fix that and have a look again.

Reply:
Fixed